CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## x.y.z
 
 ### Added
+- allow ungeristration_notice in thankyouemail
```

Comment:
I'd say this is more of a bug fix than a new feature and hence should go to the "fixed" section. What do you think?

Also, please use the same format for the changelog entry as the other existing entries.

Reply:
Done.